joomla-projects/privacy-framework
# Detect when sending email is disbled and adjust accordingly

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/21646

### Summary of Changes

When sending mail is disabled on a site, we cannot process information requests.  Therefore, the UI is adjusted to:

- Show a warning on the frontend form saying that requests cannot be created
- Hide the "New" button in the admin list view
- Fail the save actions in the models when creating new requests
- Force a redirect in the admin when trying to navigate to the request view's edit layout (create a request)
- Hide the button to email the data export to users on the list and item views

### Testing Instructions

Try to create an information request when mail sending is disabled.  You should be met with warnings and an inability to create the request.
